doc/reference/metrics.md

```diff
@@ -0,0 +1,59 @@
+# Prometheus
+
+- [Prometheus](#prometheus)
+    - [Exporter URI](#exporter-uri)
+    - [Metrics](#metrics)
+      - [Objects](#objects)
+        - [HTTPServer](#httpserver)
+      - [Filters](#filters)
+        - [Proxy](#proxy)
+    - [Create Metrics for Extended Objects and Filters](#create-metrics-for-extended-objects-and-filters)
+
+Easegress has a builtin Prometheus exporter.
+
+## Exporter URI
+
+```
+Get /apis/v2/metrics
+```
+
+## Metrics
+
+### Objects
+
+#### HTTPServer
+
+| Metric | Type | Description | Labels                                                                  |
+|--------| ---- | ----------- |-------------------------------------------------------------------------|
+| httpserver_health | gauge | show the status for the http server: 1 for ready, 0 for down | clusterName, clusterRole, instanceName, name, kind                      |
+| httpserver_total_requests | counter | the total count of http requests | clusterName, clusterRole, instanceName, name, kind, routerKind, backend |
+| httpserver_total_response | counter | the total count of http resposne | clusterName, clusterRole, instanceName, name, kind, routerKind, backend |
+| httpserver_total_error_requests | counter | the total count of http error requests | clusterName, clusterRole, instanceName, name, kind, routerKind, backend |
+| httpserver_requests_duration | histogram | request processing duration histogram | clusterName, clusterRole, instanceName, name, kind, routerKind, backend |
+| httpserver_requests_size_bytes | histogram | a histogram of the total size of the request. Includes body | clusterName, clusterRole, instanceName, name, kind, routerKind, backend |
+| httpserver_response_size_bytes | histogram | a histogram of the total size of the returned response body | clusterName, clusterRole, instanceName, name, kind, routerKind, backend |
```

Comment:
```suggestion
| httpserver_responses_size_bytes | histogram | a histogram of the total size of the returned response body | clusterName, clusterRole, instanceName, name, kind, routerKind, backend |
```

and the `description` is different from `httpserver_requests_size_bytes`, is it correct?